examples/nidcpower_source_dc_voltage/_helpers.py

```diff
@@ -48,23 +48,26 @@ def __init__(self, *, grpc_channel: grpc.Channel):
             pin_map_service_pb2_grpc.PinMapServiceStub(grpc_channel)
         )
 
-    def update_pin_map(self, pin_map_id: str) -> None:
+    def update_pin_map(self, pin_map_path: str) -> str:
         """Update registered pin map contents.
 
         Create and register a pin map if a pin map resource for the specified pin map id is not
         found.
 
         Args:
-            pin_map_id (str): The resource id of the pin map to register as a pin map resource. By
+            pin_map_path (str): The file path of the pin map to register as a pin map resource. By
                 convention, the pin map id is the .pinmap file path.
 
+        Returns:
+            str: Specifies the registered pin map id.
+
         """
-        pin_map_path = pathlib.Path(pin_map_id)
+        pin_map_purepath = pathlib.Path(pin_map_path)
```

Comment:
This is not a pathlib.PurePath. pathlib.PurePath is like pathlib.Path except it doesn't support filesystem access, and this function performs filesystem access.

Reply:
Fixed

Reply:
Renamed